evennia/settings_default.py

```diff
@@ -102,6 +102,9 @@
 # the client will itself figure out this url based on the server's hostname.
 # e.g. ws://external.example.com or wss://external.example.com:443
 WEBSOCKET_CLIENT_URL = None
+# Number of trusted web servers (reverse proxies) in front of this server which
+# set the X-Forwarded-For header.
+WEBSOCKET_TRUST_X_FORWARDED_FOR = None
```

Comment:
I think it would be good to give a comment with a short example here,  of what setting to put here for an example proxy chain. This is after all not immediately intuitive (as exemplified by the questions from friarzen).